src/components/App.tsx

```diff
@@ -747,6 +747,10 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         this.initializeScene();
       });
     }
+
+    if (window.location.search.includes("share-target")) {
```

Comment:
Let’s use the proper API here: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/has